request_a_govuk_domain/request/validators.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, error_message=None):
 
 
 class PhoneNumberValidator(BaseValidator):
-    phone_number_pattern = re.compile(r"^\s*\d(?:\s*\d){10}\s*$")
```

Comment:
I think the regex is incorrect?

Reply:
It is correct but the commit was unclear that it's 10 or 11 digits, including the leading 0, that makes a valid phone number. I'm changing the regex so the leading digit is always 0.